buildozer/default.spec

```diff
@@ -208,6 +205,12 @@ android.arch = armeabi-v7a
 # Python for android (p4a) specific
 #
 
+# (str) python-for-android fork to use, defaults to upstream (kivy)
+#p4a.fork = kivy
```

Comment:
Minor remark: so for now, we're assuming the fork is on github only right?

Reply:
Yes, and also we assume that the name of the fork has not changed :wink:

Reply:
That's more than enough for a first iteration and that probably covers more than 80% of the cases